webrtc.html

```diff
@@ -4461,6 +4461,12 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
     is designed to minimize occurrence of images with with letter
     box or or pillow boxing. The media MUST NOT be upscaled to
     create fake data that did not occur in the input source.</p>
+    <p>When video is rescaled, for example for certain combinations
+    of width or height and
+    <code><a data-link-for="RTCRtpEncodingParameters">
+    scaleResolutionDownBy</a></code> values, situations when the result must
+    be rounded may occur. In such situations the user agent is allowed to
```

Comment:
I think we should be less wishy washy here and say how it must round - basically we can choose from floor(), ceil(), or round(). It should not be different on different browsers.

Reply:
Perhaps that's what we should do. I made this a bit open since I was unsure if that would cause problems (perhaps 71 pixels as in the example in #1564 is hard to do, while 70 or 68 is easy?). Surely sending a video with width 70 (or even 68) is preferred over not sending when the division gives 71.3333333, so I gave some room for implementations. But if we do not foresee that being a problem I totally agree to your comment.

Reply:
All the specs for any video codec anyone uses any more work fine with 71 and though there is rumor of some implementations that might have bugs at for sizes that are not even, it seems  those are largely fixed so I really don't see this as a issue any more. I think we would be fine to say it gets rounded to 71. I think my vote would be for "round up to nearest integer"